pkg/automationconfig/automation_config_builder.go

```diff
@@ -214,11 +213,6 @@ func (b *Builder) Build() (AutomationConfig, error) {
 			Version:                     b.mongodbVersion,
 			AuthSchemaVersion:           5,
 		}
-		process.SetSystemLog(SystemLog{
-			Destination: "file",
-			Path:        path.Join(DefaultAgentLogPath, "/mongodb.log"),
-			LogAppend:   true,
-		})
```

Comment:
By removing this `mongodb` now by defaults sends its output to stdout: https://docs.mongodb.com/manual/reference/configuration-options/#mongodb-setting-systemLog.destination